packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/CameraZoom.java

```diff
@@ -1,52 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I agree that this and its corresponding test should be deleted, since this doesn't seem to be used anywhere. @stuartmorgan can you double check me on this? I'm not sure if you have more context.

Reply:
Looks like it was replaced with `ZoomLevelFeature` during a refactor and the dead code just accidentally wasn't cleaned up.